### PR TITLE
Add GitHub organization documentation

### PR DESCRIPTION
**Pull request by jflory7**
This PR adds a new page for the GitHub organization. It documents a bit of how we use it and also why we do things in a specific way.

Please merge this after **+2 approved reviews**.

---

_Tracking RITlug/tasks#8_

### CODE REVIEW

File: docs/administration/github-org.rst [hunk 1]
```diff
@@ -1,10 +1,102 @@
...
-.. _`RITlug GitHub organization`: https://github.com/RITlug
 
-…more to come soon.
+*******
+Purpose
+*******
+
+The GitHub organization is a central place to manage club projects, an
+approximation of membership, and host some of our infrastructure. Most data is
+static and doesn't change often. The GitHub organization is usually relevant by
```
Comment (Tjzabel) [inline docs/administration/github-org.rst]: Not entirely sure what you mean by "GitHub org is usually relevant by individual club projects..." The wording is a little off to me.

File: docs/administration/github-org.rst [hunk 2]
```diff
@@ -1,10 +1,102 @@
...
 
-.. _`RITlug GitHub organization`: https://github.com/RITlug
 
-…more to come soon.
+*******
+Purpose
+*******
+
+The GitHub organization is a central place to manage club projects, an
+approximation of membership, and host some of our infrastructure. Most data is
```
Comment (Tjzabel) [inline docs/administration/github-org.rst]: Change wording a little.

"The GitHub organization is a central place to manage club projects, membership, and infrastructure."

This allows more concise wording. If you want any other changes feel free to ping me.

**Comment by jflory7** [comment]
@Tjzabel Thanks, really subtle but really helpful feedback. I reworded. Could you please re-review?

File: docs/administration/github-org.rst [hunk 3]
```diff
@@ -1,10 +1,102 @@
...
+
+#. **Eboard**: Club leadership (grants admin access to all repositories)
+
+#. **Friends of RITlug**: Active participants with the club from outside RIT or
+   other friends of the club (does not include permissions)
+
+#. **Members**: RIT students, faculty, and staff participating in RITlug (does
+   not include permissions)
+
+#. **TigerOS Team**: `TigerOS`_ project team (grants admin access to TigerOS
```
Comment (axk4545) [inline docs/administration/github-org.rst]: @jflory7 This is actually incorrect. There is a separate TigerOS Admins team for the. The TigerOS Team grants write permission.

**Comment by jflory7** [comment]
@axk4545 Thanks, I pushed a new commit to address your feedback.

File: docs/administration/github-org.rst [hunk 4]
```diff
@@ -1,10 +1,102 @@
...
+before officially endorsing it underneath our GitHub organization.
+
+Generally, repository creation should be encouraged rather than discouraged.
+
+
+******************
+Adding new members
+******************
+
+There is no criteria for adding new members to the RITlug GitHub organization.
```
Comment (ct-martin) [inline docs/administration/github-org.rst]: Really? This also contradicts the "active community members" on L39.

Comment (jflory7) [reply docs/administration/github-org.rst]: @ct-martin When I said we don't have criteria for adding new members, we do not have a firm set of criteria, e.g. attend X number of meetings, participate over X amount of time, etc.

What change are you suggesting?

Comment (ct-martin) [reply docs/administration/github-org.rst]: @jflory7 At the very least, not saying there are no criteria. If I remember correctly, the "technical" definition for Club Office is attending three meetings/semester (correct me if I'm wrong). While this may seem like an unlikely scenario, what we don't want to have happen is someone who none of us have ever interacted with coming and waving this in our face.

Comment (jflory7) [reply docs/administration/github-org.rst]: @ct-martin I think this fits into a larger discussion for our club membership. I am not aware of any existing guideline / requirement even to club office for our membership. As it is now, there really is not any criteria. I only add people when I remember.

We can move a membership discussion to [RITlug/tasks](https://github.com/RITlug/tasks), but I don't think this should block this PR. We can go back and make a change after we consider general membership criteria.

Personally, I would like club members to ask to be included in the GitHub organization. In your scenario you gave, we can use our own discretion for how to answer. If someone who we never see asks to be included in our GitHub organization, I would not consider them an active community member and would use that as my reply.

Comment (ct-martin) [reply docs/administration/github-org.rst]: @jflory7 In that case, I would put being an active community member as the criteria, or just drop the line in favor of L39, "Active community members of RITlug should be added to the GitHub organization."
I'm going to maintain my blocking review and cite your response to my scenario as additional reasoning for why.

File: docs/administration/github-org.rst [hunk 5]
```diff
@@ -1,10 +1,103 @@
...
+There are no requirements to add new members to the RITlug GitHub organization
+other than being an active community member. In the future, if a membership
+criteria is developed, this section should be updated.
+
+Active community members of RITlug should be added to the GitHub organization.
```
Comment (ct-martin) [inline docs/administration/github-org.rst]: With the new commit, this becomes redundant. Maybe a horizontal line if you don't want to keep it like this? I'm going to dismiss my blocking review, but I'm not going to approve the PR since this feels awkward to me.

Comment (ct-martin) [reply docs/administration/github-org.rst]: @jflory7 ^

Comment (jflory7) [reply docs/administration/github-org.rst]: I think it's okay. It's important we keep good habits of keeping things well-maintained over time – commit early, commit often. If it's not perfect now, we can always revisit. I think a membership discussion is more important to clarifying this, but it won't be solved relatively soon to block this PR.
